cmd/snap/cmd_debug_timings.go

```diff
@@ -42,6 +44,8 @@ func init() {
 		func() flags.Commander {
 			return &cmdChangeTimings{}
 		}, changeIDMixinOptDesc.also(map[string]string{
+			"ensure": i18n.G("Show timings for a change related to the given Ensure activity (one of: auto-refresh, become-operational, refresh-catalogs, refresh-hints, seed)"),
```

Comment:
It's a debug command, so not a high priority, but I wonder whether we could take some steps to make sure the list gets updated when new `ensure` tags are added.